src/ppx/graphics_util.cpp

```diff
@@ -767,7 +768,7 @@ Result CreateImageFromFile(
     else {
         ppxres = Result::ERROR_IMAGE_FILE_LOAD_FAILED;
     }
-
+    
```

Comment:
avoid spaces/line breaks changes if possible, the original code might have its own style